code/style.css

```diff
@@ -1,4 +1,249 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/****html {
+  background: #7886d6;
+  color: rgb(42, 12, 92);
+}
+******/
+
+body {
+  background: #7886d6;
+  color: rgb(42, 12, 92);
+}
+
+/***************   HEADER    ************/
+
+header {
+    height: 60px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 15px; 
+    
+   }
+
+h2 {
+  padding-left: 20px;
+  padding-top: 10px;
+  font-family: Helvetica; 
+  font-size: 15px;
+  color: rgb(36, 233, 10);
+  width: 30%;
+}
+
+header img {
+    width: 50px;
+    padding-left: 10px;
+}
+
+nav {
+  padding: 20px;
+   
+}
+
+nav a {
+    color: #fff;
+    font-size: 10px;
+    text-decoration: none;
+    margin-left: 10px;
+  }
+
+/* NAV bar*/
+.sticky {
+  
+  position: fixed;
+  width: 92%;
+  background: navy;
+  display: flex;
+  justify-content: right;
+  border: 2px solid rgb(74, 250, 5);
+  box-shadow: 2px 5px 2px 2px rgba(34, 33, 33, 0.4);
+  }
+
+  
+/**************  MAIN NEWS     ****************/
+
+.big-news-container {
+    padding: 100px, 10px;
+    margin: 8px;
+    display: flex;
+    flex-direction: column;
+  }
+
+.big-news-container img {
+    width: 100%;
+    border: 2px solid black;
+    box-shadow: 2px 5px 2px 2px rgba(34, 33, 33, 0.4);
+
+}
```

Comment:
This image is really big! A suggestion is to crop it, using a fixed height together with `object-fit: cover;`